src/pages/Cart/Cart.js

```diff
@@ -1,78 +1,147 @@
 import React, { Component } from 'react';
+import { API } from '../../config';
 import './Cart.scss';
+import CartList from './CartList';
 
 export class Cart extends Component {
+  state = {
+    cartData: '',
+  };
+
+  quantityPlus = (cartIndex, cartItemId) => {
+    let newCartData = [...this.state.cartData];
+    if (newCartData[cartIndex - 1].quantity < 100) {
+      newCartData[cartIndex - 1].quantity =
+        newCartData[cartIndex - 1].quantity + 1;
+      this.setState({ cartData: newCartData });
+    } else {
+      this.setState({ cartData: newCartData });
+    }
```

Comment:
- 조금 반복적인 것 같아서, 얼리 리턴 패턴을 사용하시면 어떨까요? 현재 코드는, 수량이 100개가 넘어서 증가가 되지 않아야 하는 경우에도 setState 와 fetch 함수를 실행합니다. 수량이 변경되지 않을 때는, 아무런 일도 발생하지 않아야 합니다!
```suggestion
    if (newCartData[cartIndex - 1].quantity >= 100) return;
    
    newCartData[cartIndex - 1].quantity = newCartData[cartIndex - 1].quantity + 1;
    this.setState({ cartData: newCartData });        
```

Reply:
오 생각만했던 부분인데 감사합니다!